scripts/benchmark_inference.py

```diff
@@ -197,13 +209,13 @@
 def one_token(model, use_cache):
     if use_cache:
         actual, _ = model.forward(
-            next_val, past_key_value_states=cache, use_cache=True, only_last_token=True
+            None, past_key_value_states=None, use_cache=True, only_last_token=True
```

Comment:
I'm not understanding what's going on in the benchmarking code here... next_val = None? kv_states=None? does regular benchmarking still work after all these changes?

Reply:
This script is not ready, and will need to be fixed. Will let you know when update is available.

Reply:
updated this script to now try default and paged attention cache